keyboards/kbdfans/kbd67/hotswap/keymaps/sanmigaro/rules.mk

```diff
@@ -0,0 +1,82 @@
+# MCU name
```

Comment:
This file should be condensed down to just:

```make
LEADER_ENABLE = yes
KEY_LOCK_ENABLE = yes
```

as the rest of these lines are unchanged from the hotswap's `rules.mk`.